recipes/xmlvalidator/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "xmlvalidator" %}
+{% set version = "1.0.0.b3" %}
+{% set sha256 = "e4eaad2a38b4e3c186e9d5f4b089e25abf517c943baffe1c1da9112a07f9f4b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.developers.cam.ac.uk/ch/hunter/ssiptools/utils/xmlvalidator/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256:  {{ sha256 }}
+
+build:
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
I don't see any C code in this package? If not, then can remove the entire `build` section and put `noarch: python` under the top-level `build`.